ts/features/it-wallet/screens/credential/ItwCredentialDetailsScreen.tsx

```diff
@@ -0,0 +1,101 @@
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { ScrollView } from "react-native-gesture-handler";
+import { SafeAreaView, View } from "react-native";
+import { constNull } from "fp-ts/lib/function";
+import ItwCredentialCard from "../../components/ItwCredentialCard";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import I18n from "../../../../i18n";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import { ItwParamsList } from "../../navigation/ItwParamsList";
+import ItwCredentialClaimsList from "../../components/ItwCredentialClaimsList";
+import { StoredCredential } from "../../store/reducers/itwCredentialsReducer";
+
+export type ItwCredentialDetailsScreenNavigationParams = {
+  credential: StoredCredential;
+};
+
+type ItwCredentialDetailscreenRouteProps = RouteProp<
+  ItwParamsList,
+  "ITW_CREDENTIAL_DETAILS"
+>;
+
+/**
+ * Renders a preview screen which displays a visual representation and the claims contained in the credential.
+ */
+const ItwCredentialDetailsScreen = () => {
+  const route = useRoute<ItwCredentialDetailscreenRouteProps>();
+  const { credential } = route.params;
+  const bannerViewRef = React.createRef<View>();
+
+  /**
+   * Content view which asks the user to confirm the issuance of the credential.
+   * @param data - the issuance result data of the credential used to display the credential.
+   */
+  const ContentView = ({ data }: { data: StoredCredential }) => {
+    const presentationButton: BlockButtonProps = {
+      type: "Solid",
+      buttonProps: {
+        label: I18n.t(
+          "features.itWallet.presentation.credentialDetails.buttons.qrCode"
+        ),
+        accessibilityLabel: I18n.t(
+          "features.itWallet.presentation.credentialDetails.buttons.qrCode"
+        ),
+        onPress: () => null
+      }
+    };
+
+    return (
+      <BaseScreenComponent goBack={true} contextualHelp={emptyContextualHelp}>
+        <SafeAreaView style={IOStyles.flex}>
+          <ScrollView contentContainerStyle={IOStyles.horizontalContentPadding}>
+            <ItwCredentialCard
+              parsedCredential={data.parsedCredential}
+              display={data.displayData}
+            />
+            <VSpacer />
+            <ItwCredentialClaimsList data={{ ...data, issuerName: "INPS" }} />
+            <VSpacer size={32} />
+            <Banner
+              testID={"ItwBannerTestID"}
+              viewRef={bannerViewRef}
+              color={"neutral"}
+              size="big"
+              title={I18n.t(
+                "features.itWallet.issuing.credentialPreviewScreen.somethingWrong.title"
+              )}
+              content={I18n.t(
+                "features.itWallet.issuing.credentialPreviewScreen.somethingWrong.subtitle",
+                {
+                  issuer: credential.parsedCredential
+                }
```

Comment:
I think the content here should be slightly different. (https://www.figma.com/file/c80PUA6BYANkGPyaqsRBNA/%5BIT-Wallet%5D---Esperienza-in-IO?type=design&node-id=7779-22160&mode=design&t=ocMXlNBP6YxfMu5q-0). Also it currently does not display the issuer data since we are not storing it with the credential but it's not really needed according to [this FIGMA].

Reply:
I'm not sure I understand. Anyway the figma you linked refers to a new version of the credential detail that we have not yet explored and which changes completely compared to the current version. I'm also not sure if there are all the components needed to make it. Related to the issuerName of the bottom banner I suggest to change locale string removing params because as figma in the support banner there are no mention about the issuer name.

Reply:
The current version is using the locale from the preview screen which has a different banner (which includes the issuer name) from the FIGMA I mentioned. Let's leave out the redesign of the whole screen, we can still just implement the banner with the new copy (which removes the issuer name) and keep the current design. The very old version doesn't have the banner at all so we either remove it completely or we can add it and use the new copy version. What do you think?

Reply:
here https://github.com/pagopa/io-app/pull/5219/commits/10660f82907e339929abae45903cf7e8f9f89797